vector/src/main/java/im/vector/util/ThemeUtils.kt

```diff
@@ -165,21 +165,14 @@ object ThemeUtils {
      * @return the resource Id for the current theme
      */
     fun getResourceId(c: Context, resourceId: Int): Int {
-        if (TextUtils.equals(getApplicationTheme(c), THEME_LIGHT_VALUE)) {
+        if (TextUtils.equals(getApplicationTheme(c), THEME_LIGHT_VALUE + THEME_STATUS_VALUE)) {
```

Comment:
Correct test is
`if (TextUtils.equals(getApplicationTheme(c), THEME_LIGHT_VALUE) || TextUtils.equals(getApplicationTheme(c), THEME_STATUS_VALUE))`

Reply:
Yes, thank you haha. I couldn't revert commit, but I guess that's because this is a PR. I'll try and do a fixed PR today after work.